dataikuapi/dss/feature_store.py

```diff
@@ -0,0 +1,14 @@
+class DSSFeatureStore(object):
+    def __init__(self, client):
+        """
+        A handle on the Feature Store. Do not call this methid directly, call """
```

Comment:
Call 911? 🤣 
The method to call is missing, also, there is a typo in `methid`.

Reply:
Ha ha. Yes, plus got the wording closer to other such cases.